app/src/main/kotlin/io/imhungry/notifications/NotificationHelper.kt

```diff
@@ -0,0 +1,97 @@
+package io.imhungry.notifications
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import io.imhungry.R
+
+class NotificationHelper(private val context: Context) {
+
+    /**
+     *  Unfortunately, NotificationCompat does actually have an ENUM behind the scenes.
+     *  This simplifies the function arguments in this class.
+     */
+    enum class Priority constructor(internal val value: Int) {
```

Comment:
Given the name of the outer class, `NotificationHelper.Priority` doesn't make much sense to me. I think that, given you are already in your own package, `Priority` should be moved into its own file. I recommend renaming it to `NotificationPriority` for the sake of being more explicit.

Reply:
Moved to another file